commit ace2f4ad883aa764a278180cba7f4f6ff1b21e23 (refs/changes/33/3233/1)
Author: Jared Morrow <jm@joyent.com>
Date:   2018-01-18T18:18:51+00:00 (1 year, 9 months ago)
    
    MANTA-3563 Muppet fails to build when cloned using public GitHub URL

diff --git a/.gitmodules b/.gitmodules
index 7b4be7d..dff00c3 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,15 +1,15 @@
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = git://github.com/davepacheco/jsstyle.git
+	url = https://github.com/davepacheco/jsstyle.git
 [submodule "deps/restdown"]
 	path = deps/restdown
-	url = git://github.com/trentm/restdown.git
+	url = https://github.com/trentm/restdown.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = git://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/manta-scripts"]
 	path = deps/manta-scripts
-	url = git@github.com:joyent/manta-scripts.git
+	url = https://github.com/joyent/manta-scripts.git
 [submodule "deps/haproxy-1.5"]
 	path = deps/haproxy-1.5
 	url = https://github.com/joyent/haproxy-1.5.git
